src/SqlPersistence/SqlPersistence.csproj

```diff
@@ -17,6 +17,16 @@
     <PackageReference Include="Particular.Packaging" Version="2.2.0" PrivateAssets="All" />
   </ItemGroup>
 
+  <ItemGroup>
```

Comment:
shouldn't there be a deleted `InternalsVisibleTo.cs` file in the PR changes?

Reply:
Fixed it for Laila